react/src/util/format.ts

```diff
@@ -1,6 +1,10 @@
 import currencyFormat from 'currency-formatter';
 import { currency } from './api-client';
 
+export const BCH_DECIMALS = 8;
+export const XEC_DECIMALS = 2;
+export const FIAT_DECIMALS = 2;
+
```

Comment:
What do you think about making this into an obj like:

`DECIMALS = { BCH: 8, XEC: 2, FIAT: 2 }`

?

Reply:
Done.